ops/model.py

```diff
@@ -47,7 +49,9 @@ class Model:
             for Kubernetes charms.
     """
 
-    def __init__(self, unit_name, meta, backend):
+    def __init__(
+            self, unit_name: str, meta: 'ops.charm.CharmMeta',
+            backend: '_ModelBackend', model_name: str = None):
```

Comment:
is it time to require keyword arguments?

Reply:
I'm happy to make model_name mandatory keyword, I don't think it is worth changing the signature for the existing parameters.

Reply:
yeah, that's what i meant (i realise how it reads the other way tho)